.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,19 +1,18 @@
 ---
-name: Bug Report
-about: A template for Druid bug reports
+name: Problem Report
+about: A template for Druid problem reports
 title: ""
-labels: Bug
 assignees: ''
 
 ---
 
 Please provide a detailed title (e.g. "Broker crashes when using TopN query with Bound filter" instead of just "Broker crashes").
 
-# Affected Version
+### Affected Version
 
 The Druid version where the bug was encountered.
```

Comment:
Let's change `bug` to `'problem` here and elsewhere, LGTM otherwise

Reply:
Thanks, changed.